components/HeaderNew/HeaderLogo.js

```diff
@@ -14,9 +16,9 @@ export default class HeaderLogo extends Component {
                 <Link
                   to={this.context.mainUrl}
                   useRawTo >
-                    <span className={styles.headerLogo} />
+                    <span className={classnames(this.context.implementationStyle['headerLogo'], getClass(styles, 'headerLogo'))} />
```

Comment:
why this.context.implementationStyle['headerLogo'] instead of this.context.implementationStyle.headerLogo ?